xml/System.Numerics/BitOperations.xml

```diff
@@ -42,10 +42,20 @@
         <Parameter Name="value" Type="System.UInt32" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The mask.</param>
+        <summary>Counts the number of leading zero bits in an unsigned 32-bit integer mask.</summary>
```

Comment:
The order of these is off. Summary is normally first...

Reply:
Not in dotnet-api-docs xml files. The format and order are different than the ones you see in xmldocs in corefx/coreclr.
The 'before' lines show you that param is first, and that's how the skeleton was automatically generated for the API by the Docs repo.